src/main/java/wolox/training/models/Book.java

```diff
@@ -116,39 +117,39 @@ public String getIsbn() {
     }
 
     private void setGenre(String genre) {
-        this.genre = genre;
+        this.genre = checkNotNull(genre);
     }
 
     private void setAuthor(String author) {
-        this.author = author;
+        this.author = checkNotNull(author);
     }
 
     private void setImage(String image) {
-        this.image = image;
+        this.image = checkNotNull(image);
     }
 
     private void setTitle(String title) {
-        this.title = title;
+        this.title = checkNotNull(title);
     }
 
     private void setSubtitle(String subtitle) {
-        this.subtitle = subtitle;
+        this.subtitle = checkNotNull(subtitle);
     }
 
     private void setPublisher(String publisher) {
-        this.publisher = publisher;
+        this.publisher = checkNotNull(publisher);
     }
 
     private void setYear(String year) {
-        this.year = year;
+        this.year = checkNotNull(year);
     }
 
     private void setPage(Integer page) {
-        this.page = page;
+        this.page = checkNotNull(page);
     }
 
     private void setIsbn(String isbn) {
-        this.isbn = isbn;
+        this.isbn = checkNotNull(isbn);
```

Comment:
After checking if it is null, you can check if the content of the isbn field is a number.